app/src/main/java/com/github/wnder/guessLocation/GuessLocationActivity.java

```diff
@@ -393,16 +385,24 @@ private void confirmButton() {
             return;
         }
 
+        //If guess has been confirmed, confirm button becomes button leading to scoreboard
+        if (guessConfirmed) {
+            //Open the scoreboard activity
+            Intent intent = new Intent(this, ScoreboardActivity.class);
+            intent.putExtra(ScoreboardActivity.EXTRA_PICTURE_ID, pictureID);
+            startActivity(intent);
+            return;
+        }
+
         if (compassMode) compassButton();
         guessConfirmed = true;
 
-        //don't show little image anymore
+        //don't show little image anymore and disable the compass button
         findViewById(R.id.imageToGuessCard).setVisibility(INVISIBLE);
+        findViewById(R.id.compassMode).setClickable(false);
```

Comment:
The compass button should be invisible to avoid confusion, if it has no use after a guess...

Reply:
increased the contrast